src/integrations/slack/utils.ts

```diff
@@ -14,4 +14,14 @@ export const isBotMessage = function (event: any): boolean {
         event.username === botConfig.BOT_NAME ||
         event.message?.username === botConfig.BOT_NAME
     );
-};
\ No newline at end of file
+};
+
+// Limit team name to 3 words
+const TEAM_NAME_WORDS_LIMIT = 3;
```

Comment:
This is to prevent abuse of the Slack team field, to affect the readability of the stats.